src/db/analytics.ts

```
@@ -61,4 +134,42 @@ export class AnalyticsDAO {
+    const now = new Date();
+    let startDate: Date;
+    let endDate: Date = now;
+
+    switch (timeWindow) {
+      case "day":
+        startDate = startOfDay(subDays(now, windowPage));
+        endDate = endOfDay(subDays(now, windowPage));
+        break;
+
+      case "week":
+        const currentWeek = subWeeks(now, windowPage);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)